website/src/assets/pages/vis5-opioid-trajectories/vis.vl.json

```diff
@@ -342,11 +341,12 @@
               "mark": {"type": "text", "fontSize": 0},
               "encoding": {
                 "color": {
-                  "title": "# Deaths",
+                  "title": "Legend",
```

Comment:
Is that right?

Reply:
I'm pretty sure that's what he meant. The gradient should say Legend since there is no interactivity.

Reply:
Ok